boot/bootdata/hivesft.inf

```diff
@@ -1484,6 +1484,7 @@ HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon","ConsoleShell",0x00
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon","Shell",0x00020000,"%SystemRoot%\explorer.exe"
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon","Userinit",0x00020000,"%SystemRoot%\system32\userinit.exe"
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon","AutoAdminLogon",0x00000000,"1"
+HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon","LogonType",0x00000000,"0"
```

Comment:
Why isn't it HKLM\Software\Microsoft\Windows\CurrentVersion\Reliability DWORD LunaUI  as described in PR ? Please fix either registry init or PR description

Reply:
I changed registry key as in XP, so Luna XP styled shut down dialog now can be activated from `HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon` with DWORD key `LogonType` to 1

Reply:
Windows Server 2003+ indeed uses the LogonType value; it doesn't use "LunaUI".